### ISSUE TITLE
About finetuning dec

### ISSUE THREAD

**Issue by harisreedhar** [tanshuai0219/EDTalk#25]
Is it fine to add multiple personalities for fine tuning? Can that increase generalization of the model?

**Comment by tanshuai0219** [comment]
> Is it fine to add multiple personalities for fine tuning? Can that increase generalization of the model?

Yep, but you should refer to [LIA dataset](https://github.com/wyhsirius/LIA/blob/db7a2e974a177eb8470e500527f940987199ad76/dataset.py#L13) to modify [finetune dataset](https://github.com/tanshuai0219/EDTalk/blob/f16c0029177391b0ba58e1ca7305f705e0108464/fine_tune/dataset.py#L12)
